src/components/commons/modal/Confirm.tsx

```diff
@@ -0,0 +1,48 @@
+import { confirmAtom } from "@stores/modal";
+import { useAtom } from "jotai";
+import ModalTextBox from "./components/ModalTextBox";
+import ModalWrapper from "./components/ModalWrapper";
+import * as S from "./components/ModalWrapper.styled";
+
+const Confirm = () => {
+  const [confirm, setConfirm] = useAtom(confirmAtom);
+  const { isOpen, title, subTitle, okText, okCallback, noText, noCallback } = confirm;
+
+  const resetConfirm = () => {
+    setConfirm({
+      isOpen: false,
+      title: "",
+      subTitle: "",
+      okCallback: () => {},
+      noCallback: () => {},
```

Comment:
p3) okText, noText도 초기화 해주세요 ~ 

```suggestion
    setConfirm({
      isOpen: false,
      title: "",
      subTitle: "",
      okText: "",
      okCallback: () => {},
      noText: "",
      noCallback: () => {},
```

Reply:
네 ~ ㅋㅋ

Reply:
네~ ㅋㅋ